pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/alderspensjon/Alderspensjon.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.pensjon.brev.maler.fraser.alderspensjon
+
+import no.nav.pensjon.brev.maler.fraser.common.Constants.KONTAKT_ENG_URL
+import no.nav.pensjon.brev.maler.fraser.common.Constants.KONTAKT_URL
+import no.nav.pensjon.brev.maler.fraser.common.Constants.NAV_URL
+import no.nav.pensjon.brev.maler.fraser.common.Constants.PENSJON_URL
+import no.nav.pensjon.brev.template.LangBokmalEnglish
+import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.Language.English
+import no.nav.pensjon.brev.template.OutlinePhrase
+import no.nav.pensjon.brev.template.dsl.OutlineOnlyScope
+import no.nav.pensjon.brev.template.dsl.text
+
+class Alderspensjon {
+    object HarDuSpoersmaal : OutlinePhrase<LangBokmalEnglish>() {
+        override fun OutlineOnlyScope<LangBokmalEnglish, Unit>.template() {
+            title1 {
+                text(
+                    Bokmal to "Har du spørsmål?",
+                    English to "Do you have questions?",
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Du finner mer informasjon på $PENSJON_URL. På $KONTAKT_URL kan du chatte eller skrive til oss. Hvis du ikke finner svar på $NAV_URL, kan du ringe oss på telefon 55 55 33 34, hverdager 09.00-15.00.",
```

Comment:
Føler at vi har skrivd samme teksten mange ganger. Kunne du sjekket om denne kan slås sammen med ett par andre tekster, eller om du kunne brukt en eksisterende frase? Evt laget noe lignende det som eksisterer i dag med Felles.xyz.[ufore/alder] hvor det kan hende kun noen konstanter må byttes ut.

Reply:
Om den ikke kan slåes sammen evt bruk eksisterende constants for tlf-nr.

Reply:
Lurt, tar ein titt.

Reply:
Kanskje det kan løses med en overload til includePhrase? Om det ikke lar seg gjøre så får vi si at dette er greit.

Reply:
Kan du ikke bruke `Felles.HarDuSpoersmaal`?

Reply:
Skulle gjerne brukt Felles.HarDuSpoersmaal, men det gir kompileringsfeil fordi denne malen berre støttar bokmål og engelsk, og da blir typeinferensen forvirra.